seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -104,7 +104,7 @@
       "id": "bd7123c9c444eddfaeb5bdef",
       "title": "Declare String Variables",
       "description": [
-        "In the previous challenge, we used the code <code>var myName = \"your name\"</code>. This is what we call a <code>String</code> variable. It is nothing more than a \"string\" of characters. JavaScript strings are always wrapped in quotes.",
+        "In the previous challenge, we used the code <code>var ourName = \"your name\"</code>. This is what we call a <code>String</code> variable. It is nothing more than a \"string\" of characters. JavaScript strings are always wrapped in quotes.",
```

Comment:
This and the translation of it does not need to be changed as the previous challenge will have exposed the user to a `myName` variable once they have completed it.